Distributed tracing 2.9 (#62410)
openshift/openshift-docs: modules/distr-tracing-tempo-install-overview.adoc

// Module included in the following assemblies:
//
// * distr_tracing_tempo/distr-tracing-tempo-installing.adoc

:_content-type: CONCEPT
[id="distr-tracing-tempo-install-overview_{context}"]
= {DTProductName} installation overview

The steps for installing {DTProductName} are as follows:

> * If your deployment strategy requires persistent storage, install the OpenShift Elasticsearch Operator via the OperatorHub.

Reviewer: ```suggestion
* Supported Log Store (AWS S3, Google Cloud Storage, Azure, Minio, OpenShift Data Foundation)
Same as Loki.
```

// Module included in the following assemblies:
//
// * distr_tracing_tempo/distr-tracing-tempo-installing.adoc

:_content-type: CONCEPT
[id="distr-tracing-tempo-install-overview_{context}"]
= {DTProductName} installation overview

The steps for installing {DTProductName} are as follows:

> * Review the documentation and determine your deployment strategy.

Reviewer: We only support one deployment strategy. I think we can remove this.

Author: Thank you, I'm removing this bullet point completely (if my understanding is correct):
```suggestion
```

* Supported Log Store (AWS S3, Google Cloud Storage, Azure, Minio, OpenShift Data Foundation)

> Same as Loki.

Reviewer: I would remove this

> * If your deployment strategy requires persistent storage, install the OpenShift Elasticsearch Operator via the OperatorHub.

Reviewer: `OpenShift Elasticsearch Operator` is not required anymore for Tempo. We can copy [this sentence from Loki](https://docs.openshift.com/container-platform/4.10/networking/network_observability/installing-operators.html#network-observability-loki-installation_network_observability):
```
* Supported Log Store (AWS S3, Google Cloud Storage, Azure, Minio, OpenShift Data Foundation)
```

Note that Swift is currently not supported for the Tempo Operator.

* Install the {TempoName} Operator via the OperatorHub.

> * Modify the custom resource YAML file to support your deployment strategy.

Reviewer: ```suggestion
* Modify the custom resource YAML file to use your custom parameters.
```
